akka-http-core/src/main/scala/akka/http/impl/settings/RoutingSettingsImpl.scala

```diff
@@ -28,6 +30,6 @@ object RoutingSettingsImpl extends SettingsCompanion[RoutingSettingsImpl]("akka.
     c getInt "range-count-limit",
     c getBytes "range-coalescing-threshold",
     c getIntBytes "decode-max-bytes-per-chunk",
-    c getString "file-io-dispatcher")
-
-}
+    c getString "file-io-dispatcher",
+    ErrorLoggingVerbosity(c getString "error-logging-verbosity"))
```

Comment:
Feels weird to reuse `ParserSettings.ErrorLoggingVerbosity` as this has no relation to parsing. Should we make `ErrorLoggingVerbosity` more general?